test/common/config/subscription_factory_test.cc

```diff
@@ -144,10 +254,10 @@ TEST_F(SubscriptionFactoryTest, HttpSubscriptionNoRefreshDelay) {
   envoy::api::v2::core::ConfigSource config;
   auto* api_config_source = config.mutable_api_config_source();
   api_config_source->set_api_type(envoy::api::v2::core::ApiConfigSource::REST);
-  api_config_source->add_cluster_names("eds_cluster");
+  api_config_source->add_cluster_names("foo");
   Upstream::ClusterManager::ClusterInfoMap cluster_map;
   Upstream::MockCluster cluster;
-  cluster_map.emplace("eds_cluster", cluster);
+  cluster_map.emplace("foo", cluster);
```

Comment:
Not sure if this lends clarity, seems a gratuitous rename.

Reply:
This isn't actually an eds cluster by default -- because it's a mock object, it defaults to the first enum in `envoy::api::v2::Cluster::`. https://github.com/envoyproxy/data-plane-api/blob/master/envoy/api/v2/cds.proto#L67